test/e2e/nightwatch.js

```diff
@@ -1,62 +1,92 @@
-      "silent" : true,
-      "output" : true,
-      "screenshots" : {
-        "enabled" : false,
-        "on_failure" : true,
-        "path" : ""
+// Default settings for local Selenium installation
+const settings = {
+  src_folders: ['./test/e2e/tests'],
+  output_folder: './reports',
+  live_output: true,
+  parallel_process_delay: 10,
+  disable_colors: false,
+  test_workers: false,
+  test_settings: {
+    default: {
+      selenium_host: 'localhost',
+      selenium_port: 4444,
+      use_ssl: false,
+      silent: true,
+      output: true,
+      screenshots: {
+        enabled: false,
+        on_failure: true,
+        path: ''
       },
-      "desiredCapabilities": {
-        "name" : "test-example",
-        "browserName": "firefox"
+      desiredCapabilities: {
+        name: 'firefox-test',
+        browserName: 'firefox'
       },
-      "globals" : {
-        "myGlobal" : "some_sauce_global"
-      },
-      "selenium" : {
-        "start_process" : false
-      }
-    },
-    "travis" : {
-      "launch_url": "https://ondemand.saucelabs.com",
-      "selenium_host" : "ondemand.saucelabs.com",
-      "selenium_port" : 443,
-      "username" : process.env.SAUCE_USERNAME,
-      "access_key" : process.env.SAUCE_ACCESS_KEY, 
-      "use_ssl" : true,
-      "silent" : true,
-      "output" : true,
-      "screenshots" : {
-        "enabled" : false,
-        "on_failure" : true,
-        "path" : ""
-      },
-      "globals": {
-        "waitForConditionTimeout": 10000
-      },
-      "desiredCapabilities": {
-        "name" : "test-example",
-        "browserName": "firefox",
-        "build": `build-${process.env.TRAVIS_JOB_NUMBER}`,
-        "tunnel-identifier": process.env.TRAVIS_JOB_NUMBER,
-      },
-      "selenium" : {
-        "start_process" : false
+      selenium: {
+        start_process: false
       }
     }
+  },
+  saucelabs_environments: []
+};
+
+// Settings for Travis CI + SauceLabs integration
+const travis = {
+  launch_url: 'https://ondemand.saucelabs.com',
+  selenium_host: 'ondemand.saucelabs.com',
+  selenium_port: 443,
+  username: process.env.SAUCE_USERNAME,
+  access_key: process.env.SAUCE_ACCESS_KEY,
+  use_ssl: true,
+  silent: true,
+  output: true,
+  globals: {
+    waitForConditionTimeout: 10000
+  },
+  selenium: {
+    start_process: false
   }
+};
+
+// Browsers that we are testing via Saucelabs
+const browsers = {
+  firefox: {
+    browserName: 'firefox'
+  },
+  chrome: {
+    browserName: 'chrome'
+  },
+  ie9: {
+    browserName: 'internet explorer',
+    version: '9'
+  },
+  ie10: {
+    browserName: 'internet explorer',
+    version: '10'
+  },
+  ie11: {
+    browserName: 'internet explorer',
+    version: '11'
+  }
+};
+
+// Merge environment settings for each browser.  This is necessary in order to test multiple browsers in parallel.
+// See: https://github.com/nightwatchjs/nightwatch-docs/blob/master/guide/running-tests/run-parallel.md
+for (var name in browsers){
```

Comment:
think this should be for ... of so that you only get the elements in the Object and won't pick up of methods.
replace var with let